### PR TITLE
MGMT-17801: Change lca-cli cmd flags into configuration file

### PR DESCRIPTION
**Pull request by tsorya**
MGMT-17801: Change lca-cli cmd flags into configuration file
    1. Adding configuration file for lca-cli command
    2. Changing iso builder to create configuration file for lca-cli

### CODE REVIEW

File: api/ibiconfig/ibiconfig.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package ibiconfig
+
+import (
+	"fmt"
+)
+
+// ImageBasedInstallConfigVersion is the version supported by this package.
+const ImageBasedInstallConfigVersion = "v1beta1"
```
Comment (tsorya) [inline api/ibiconfig/ibiconfig.go:8]: not sure i need it, though to use it for installer

File: ib-cli/installationiso/installationiso.go [hunk 2]
```diff
@@ -56,22 +54,23 @@ const (
 	rhcosIsoFileName          = "rhcos-live.x86_64.iso"
 	ibiIsoFileName            = "rhcos-ibi.iso"
 	coreosInstallerImage      = "quay.io/coreos/coreos-installer:latest"
+	ibiConfigFileName         = "ibiConfiguration"
```
Comment (eranco74) [inline ib-cli/installationiso/installationiso.go:57]:
```suggestion
	ibiConfigFileName         = "ibi-configuration.json"
```

File: ib-cli/installationiso/installationiso.go [hunk 3]
```diff
@@ -21,19 +23,15 @@ type InstallationIso struct {
...
-	SkipDiskCleanup     bool
+	AuthFilePath         string
+	PullSecretPath       string
+	IBIConfigurationPath string
+	BackupSecret         string
+	PullSecret           string
+	SshPublicKey         string
+	InstallSeedScript    string
+	SeedImage            string
+	IBIConfiguration     string
```
[the remark this answers is not in the document]
Comment (tsorya) [reply ib-cli/installationiso/installationiso.go:34]: yep but they are also part of ignition data as we need them to create butane files

**Review by eranco74 — COMMENTED** [review]
/lgtm
I think it will be usefull to create a separation between generating the ignition and generating the ISO, so later we can consume the ignition generation  from the openshift-installer

**Pull request merged**